<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>981</width>
    <height>563</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Insertar viaje fuera de la ciudad</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QPushButton#btn_add:hover{
background-color: rgb(85, 170, 0);
}
QPushButton#btn_add{
color: rgb(0, 0, 0);
font: italic 12pt &quot;Poor Richard&quot;;
background-color: rgb(255, 255, 127);
border-radius: 5px;
}

QPushButton#btn_upt:hover{
background-color: rgb(85, 170, 0);
}
QPushButton#btn_upt{
color: rgb(0, 0, 0);
font: italic 12pt &quot;Poor Richard&quot;;
background-color: rgb(255, 255, 127);
border-radius: 5px;
}

QPushButton#btn_del:hover{
background-color: rgb(85, 170, 0);
}
QPushButton#btn_del{
color: rgb(0, 0, 0);
font: italic 12pt &quot;Poor Richard&quot;;
background-color: rgb(255, 255, 127);
border-radius: 5px;
}

QPushButton#btn_close:hover{
background-color: rgb(85, 170, 0);
}
QPushButton#btn_close{
color: rgb(0, 0, 0);
font: italic 12pt &quot;Poor Richard&quot;;
background-color: rgb(255, 255, 127);
border-radius: 5px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" colspan="2">
    <widget class="QWidget" name="widget_3" native="true">
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0" rowspan="2">
       <widget class="QGroupBox" name="groupBox">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: italic 13pt &quot;Poor Richard&quot;;</string>
        </property>
        <property name="title">
         <string>Características del viaje fuera de la ciudad</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Códido de viaje</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1" rowspan="5">
          <widget class="QWidget" name="widget" native="true"/>
         </item>
         <item row="0" column="2">
          <widget class="QLineEdit" name="txt_codigo">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Fecha de inicio</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QDateTimeEdit" name="date_fecha_inicio">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2018</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
             <year>2050</year>
             <month>12</month>
             <day>31</day>
            </datetime>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2000</year>
             <month>9</month>
             <day>14</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Fehca de terminación</string>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QDateTimeEdit" name="date_fecha_termina">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="dateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2018</year>
             <month>1</month>
             <day>1</day>
            </datetime>
           </property>
           <property name="maximumDateTime">
            <datetime>
             <hour>23</hour>
             <minute>59</minute>
             <second>59</second>
             <year>2050</year>
             <month>12</month>
             <day>31</day>
            </datetime>
           </property>
           <property name="minimumDateTime">
            <datetime>
             <hour>0</hour>
             <minute>0</minute>
             <second>0</second>
             <year>2000</year>
             <month>9</month>
             <day>14</day>
            </datetime>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Número de taxi</string>
           </property>
          </widget>
         </item>
         <item row="3" column="2">
          <widget class="QSpinBox" name="spn_numero_taxi">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_5">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Distancia en Klm</string>
           </property>
          </widget>
         </item>
         <item row="4" column="2">
          <widget class="QSpinBox" name="spn_distancia">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="minimum">
            <number>0</number>
           </property>
           <property name="maximum">
            <number>10000</number>
           </property>
           <property name="value">
            <number>0</number>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QCheckBox" name="check_habilitar">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 5px;
</string>
           </property>
           <property name="text">
            <string>Chofer tuvo que habilitar?</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QGroupBox" name="groupBox_2">
        <property name="styleSheet">
         <string notr="true">color: rgb(255, 255, 255);
font: italic 13pt &quot;Poor Richard&quot;;</string>
        </property>
        <property name="title">
         <string>Características del chofer</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_6">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Nombre</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="txt_nombre">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_7">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Sexo</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QWidget" name="widget_2" native="true">
           <layout class="QGridLayout" name="gridLayout_2">
            <item row="0" column="0">
             <widget class="QRadioButton" name="rb_mas">
              <property name="text">
               <string>M</string>
              </property>
              <property name="checked">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QRadioButton" name="rb_fem">
              <property name="text">
               <string>F</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_8">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Edad</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QSpinBox" name="spn_edad">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="minimum">
            <number>18</number>
           </property>
           <property name="maximum">
            <number>140</number>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_9">
           <property name="styleSheet">
            <string notr="true">font: italic 12pt &quot;Poor Richard&quot;;
	color: rgb(255, 255, 255);
border-radius: 10px;
</string>
           </property>
           <property name="text">
            <string>Experiencia</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QSpinBox" name="spn_exp">
           <property name="styleSheet">
            <string notr="true">color: rgb(0, 0, 0);</string>
           </property>
           <property name="minimum">
            <number>1</number>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QWidget" name="widget_4" native="true"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QTableWidget" name="tabla">
     <property name="styleSheet">
      <string notr="true">color: rgb(0, 0, 0);
border-radius: 30px;
font: italic 12pt &quot;Poor Richard&quot;;</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QWidget" name="widget_6" native="true"/>
   </item>
   <item row="2" column="1">
    <widget class="QWidget" name="widget_5" native="true">
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <widget class="QPushButton" name="btn_add">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Agregar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btn_upt">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Actualizar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="btn_del">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Eliminar</string>
        </property>
       </widget>
      </item>
      <item row="0" column="3">
       <widget class="QPushButton" name="btn_close">
        <property name="cursor">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="styleSheet">
         <string notr="true"/>
        </property>
        <property name="text">
         <string>Cerrar</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
